packages/atomic/src/components/commerce/atomic-product-section-actions/atomic-product-section-actions.new.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {wrapInCommerceInterface} from '@/storybook-utils/commerce/commerce-interface-wrapper';
+import {wrapInCommerceProductList} from '@/storybook-utils/commerce/commerce-product-list-wrapper';
+import {wrapInProductTemplate} from '@/storybook-utils/commerce/commerce-product-template-wrapper';
+import {parameters} from '@/storybook-utils/common/common-meta-parameters';
+import {renderComponent} from '@/storybook-utils/common/render-component';
+
+const {
+  decorator: commerceInterfaceDecorator,
+  play: initializeCommerceInterface,
+} = wrapInCommerceInterface({
```

Comment:
We could just call `wrapInCommerceInterface` with no argument here I think:

- skipFirstRequest is treated as false by default
- default type is search, but it shouldn't matter for these stories
- default engine config should do the job

Same comment for each atomic-product-section-* .new.stories file